core/src/language/swift.rs

```diff
@@ -529,12 +530,17 @@ impl Swift {
                                 "
 			case .{case_name}:
 				if let content = try? container.decode({case_type}.self, forKey: .{content_key}) {{
```

Comment:
I believe this will fail if content is null. I think we need an else if case to capture that.

```
			case .{case_name}:
				if let content = try? container.decode({case_types}.self, forKey: .content) {
					self = .{case_name}(content)
					return
				}
				else if let isNil = try? container.decodeNil(forKey: .content), isNil {
					self = .{case_name}(nil)
					return
				} 

```

Reply:
Should be fixed in the latest commit.